### PR TITLE
Vb 1536 proactively manage bookings, added code to create dto's and then call the visit scheduler client

### PR DESCRIPTION
**Pull request by AledWEvans**
Vb 1536 proactively manage bookings, added code to create dto's and then call the visit scheduler client

### CODE REVIEW

**Review by dhirajshettymoj — DISMISSED** [review]
LGTM

File: helm_deploy/values-prod.yaml [hunk 1]
```diff
@@ -24,3 +24,7 @@ generic-prometheus-alerts:
     - "book-a-prison-visit-prod-hmpps_prison_visits_event_dlq"
   sqsAlertsOldestThreshold: 1
   sqsAlertsTotalMessagesThreshold: 1
+
+hmpps:
```
Comment (dhirajshettymoj) [inline helm_deploy/values-prod.yaml:28]: should this be HMMPS_SQS_ENABLED and be part of env (line 19)?

File: helm_deploy/values-prod.yaml [hunk 2]
```diff
@@ -24,3 +24,7 @@ generic-prometheus-alerts:
...
+  sqs:
+    enabled: false
```
Comment (dhirajshettymoj) [inline helm_deploy/values-prod.yaml:30]: also should this be added to application.yml as well for other envs?

File: helm_deploy/values-prod.yaml (hunk 1 above)
Comment (AledWEvans) [reply helm_deploy/values-prod.yaml:28]: Yeha you right, should be in env...

**Pull request merged**